content/issues/tracking-your-work-with-issues/about-task-lists.md

```diff
@@ -55,14 +55,19 @@ If you add a task list to the body of an issue, the list has added functionality
 
 You can reorder the items in a task list by clicking to the left of a task's checkbox, dragging the task to a new location, and dropping the task. You can reorder tasks across different lists in the same comment, but you can not reorder tasks across different comments.
 
-![Reordered task list](/assets/images/help/writing/task-list-reordered.gif)
+{% ifversion fpt %} ![Reordered task list](/assets/images/help/writing/task-list-reordered.gif)
+{% else %} ![Reordered task list](/assets/images/enterprise/writing/task-list-reorder.gif) {% endif %}
```

Comment:
https://docs-10528--fix-task-lists.herokuapp.com/en/enterprise-server@3.2/issues/tracking-your-work-with-issues/about-task-lists shows a broken image. Looks as if this is a tiny typo.

```suggestion
{% else %} ![Reordered task list](/assets/images/enterprise/writing/task-lists-reorder.gif) {% endif %}
```